services/ipservice.go

```diff
@@ -0,0 +1,35 @@
+package services
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"strings"
+)
+
+func GetPublicIP() (string, error) {
+	services := []string{
+		"https://api.ipify.org",
+		"https://ifconfig.me",
+		"https://ipinfo.io/ip",
```

Comment:
We should probably have these in the `.env` file?

Reply:
I want to tidy this up a bit in the future, maybe with a few more `IP_SERVICE` additions in the `.env` file or something and iterate through them but for now I think it's ok- LMK